google/cloud/internal/external_account_token_source_aws.cc

```diff
@@ -100,19 +100,45 @@ StatusOr<ExternalAccountTokenSourceAwsInfo> ParseExternalAccountTokenSourceAws(
   auto url = ValidateStringField(credentials_source, "url",
                                  "credentials-source", kDefaultUrl, ec);
   if (!url) return std::move(url).status();
-  auto regional_url =
+  auto verification_url =
       ValidateStringField(credentials_source, "regional_cred_verification_url",
                           "credentials-source", ec);
-  if (!regional_url) return std::move(regional_url).status();
+  if (!verification_url) return std::move(verification_url).status();
   auto imdsv2 =
       ValidateStringField(credentials_source, "imdsv2_session_token_url",
                           "credentials-source", std::string{}, ec);
   if (!imdsv2) return std::move(imdsv2).status();
+
+  auto invalid_url = [](absl::string_view name) {
+    return absl::StrCat("the `", name,
+                        "` field should refer to the AWS metadata service");
```

Comment:
Should we include the failing value in the message?

Reply:
Done.